crates/store/src/db/sql/mod.rs

```diff
@@ -200,6 +198,239 @@ pub fn select_accounts_by_ids(
     Ok(result)
 }
 
+/// Computes public account states for the particular block number using the given [Connection].
+///
+/// # Returns
+///
+/// Account states vector, or an error.
+#[allow(dead_code)] // TODO: remove once the function is used
```

Comment:
Or remove if no longer unused.

```suggestion
#[expect(dead_code)] // TODO: remove once the function is used
```